packages/vsx-registry/src/browser/vsx-extensions-model.ts

```diff
@@ -86,7 +86,8 @@ export class VSXExtensionsModel {
 
     protected _installed = new Set<string>();
     get installed(): IterableIterator<string> {
-        return this._installed.values();
+        const items = Array.from(this._installed).sort((a, b) => this.compareExtensions(a, b));
```

Comment:
It has to be sorted then it is assigned, not on each access

Reply:
> It has to be sorted once when it is assigned, not on each read access

You're right, I'll try to find the appropriate place where to perform the sorting.